plugins/woocommerce-blocks/assets/js/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -18,23 +17,21 @@
 	width: unset;
 	.variations_button,
 	form.cart {
-		--whole-width-in-grid: span 9999; // A large number to span the whole width of the grid.
 		&::before {
-			content: " ";
-			grid-column: var(--whole-width-in-grid);
+			grid-column: 1 / 3;
 		}
```

Comment:
Do we need `&::before` selector?

Reply:
The `&::before` element is added via legacy CSS:

https://github.com/woocommerce/woocommerce/blob/40695875f858134024d71adb4f5a4ad6ff8d70a6/plugins/woocommerce/client/legacy/css/woocommerce.scss#L369-L480


So, we need to set the right position in the grid.